taskqueue/tests/integration/conftest.py

```diff
@@ -1,19 +1,54 @@
 import os
 
 import pytest
+import aiohttp
+
+from gcloud.aio.taskqueue import PullQueue, PushQueue
+
+
+PROJECT = 'voiceai-staging'
+CREDS = os.environ['GOOGLE_APPLICATION_CREDENTIALS']
+PULL_QUEUE_NAME = 'public-test'
+PUSH_QUEUE_NAME = 'public-test-push'
 
 
 @pytest.fixture(scope='module')  # type: ignore
 def creds() -> str:
     # TODO: bundle public creds into this repo
-    return os.environ['GOOGLE_APPLICATION_CREDENTIALS']
+    return CREDS
 
 
 @pytest.fixture(scope='module')  # type: ignore
 def project() -> str:
-    return 'voiceai-staging'
+    return PROJECT
 
 
 @pytest.fixture(scope='module')  # type: ignore
-def task_queue() -> str:
-    return 'public-test'
+def pull_queue_name() -> str:
+    return PULL_QUEUE_NAME
+
+
+@pytest.yield_fixture(scope='function')  # type: ignore
+async def pull_queue_context():
+    # main purpose is to be do proper teardown of tasks created by tests
+    async with aiohttp.ClientSession() as session:
+        tq = PullQueue(PROJECT, CREDS, PULL_QUEUE_NAME, session=session)
```

Comment:
Rather than duplicating the constants here, just use the test fixture which returns them. That will allow us to use fixture overrides rather than forcing these to be globally constant.

Reply:
good point!